test/runnable/e7804.d

```diff
@@ -94,6 +96,47 @@ struct UnitTests
     }
 }
 
+
+class One
+{
+    void foo(){}
+    void foo(int){}
+}
+
+class Two : One
+{
+    void test()
+    {
+        alias Seq(T...) = T;
+        alias p1 = Seq!(__traits(getMember, super, "foo"))[0];
+        alias p2 = __traits(getMember, super, "foo");
+        static assert(__traits(isSame, p1, p2));
+    }
+}
+
+
+class SingleSymTuple
+{
+    int foo(){return 42;}
+    void test()
+    {
+        alias f = __traits(getMember, this, "foo");
+        assert(f() == 42);
```

Comment:
> Error: function expected before `()`, not `tuple(this.foo)` of type `(int())`

Some kind of extraction is needed to fix entirely the problem:

Reply:
There was a helper func in traits for that actually. No need to create a tuple, no need to call checkType either.